spec/ttml1.xml

```diff
@@ -4716,15 +4716,43 @@ as a style property only to those element types indicated in the following table
 </tbody>
 </table>
 <p>If the value of this attribute is <code>normal</code>, then the
-computed value of the style property must be considered to be no less than the
-size of the largest font size that applies to the element and its
-descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>.</p>
+resolved computed value of this style property should be determined as follows:</p>
```

Comment:
TTML2 was intended to include the `special inheritance semantics` section, but it was missed before CR2, so AFAIK the TTML1 text is preferred.

I can revert "resolved computed value" to "computed value" to match CR1 if you prefer.

Reply:
That would be factually wrong, i.e., to use _computed value_, because the computed value is the inherited value, but the result of this computation is not the inherited value.

Reply:
My reading of the TTML2 language is that it bypasses the need for a special section on inheritance and that as it stands in this pull request the special inheritance semantics don't apply.

Reply:
@nigelmegitt The following note was added to TTML2 instead of the special inheritance section. I suggest we keep the normative special inheritance section.

_When the specified value of this attribute is normal, the computed value is the specified value; therefore, when applying inheritance semantics, the value normal is the inherited value. However, when this computed value is to be used to perform paragraph layout, it is further processed to obtain a resolved computed value (also called a used value) as described above._